template/en/default/global/setting-descs.none.tmpl

```diff
@@ -56,6 +56,7 @@
    "possible_duplicates"              => "Display possible duplicates when reporting a new $terms.bug",
    "requestee_cc"                     => "Automatically add me to the CC list of $terms.bugs I am requested to review",
    "api_key_only"                     => "Require API key authentication for API requests",
+   "use_elasticsearch"                => "Use elasticsearch for $terms.bug searches when possible",
```

Comment:
Excellent questions! it is opt-in for our users because we think is a good idea to not harm people's productivity by making search behave slightly different (if faster).
For logged-out users, we make it opt-out because less time will be spent on queries.

Discoverability of this user preference is not very good (but no worse than the other parameters), however we do advertize that elasticsearch is in use and provide a way to disable it on a per-query basis already -- this is visible on bugzilla-dev.allizom.org right now.